### ISSUE TITLE
Cant delete model

### ISSUE THREAD

**Issue by ToasterUwU** [manyfold3d/manyfold#1790]
Also a new thing after the update, title says it all, log is below:

```
I, [2024-01-30T20:57:13.430552 #84]  INFO -- : [d32d9a9a-fd5f-470e-b962-0c9f01dc0e48] method=DELETE path=/libraries/2/models/69 format=html controller=ModelsController action=destroy status=500 allocations=2062 duration=21.40 view=0.00 db=4.59
F, [2024-01-30T20:57:13.432470 #84] FATAL -- : [d32d9a9a-fd5f-470e-b962-0c9f01dc0e48]   
[d32d9a9a-fd5f-470e-b962-0c9f01dc0e48] Pundit::NotDefinedError (unable to find policy `NilClassPolicy` for `nil`):
[d32d9a9a-fd5f-470e-b962-0c9f01dc0e48]   
[d32d9a9a-fd5f-470e-b962-0c9f01dc0e48] app/controllers/models_controller.rb:91:in `destroy'
```

**Comment by ToasterUwU** [comment]
Just in case, i tried adding a license and then deleting the model, just to see if its related to #1778, but that didn't change anything, so its a totally separate thing.

**Comment by Floppy** [comment]
Thanks! On it.

**Comment by Floppy** [comment]
Replicated this one locally, fixing now.

**Comment by Floppy** [comment]
This should now be fixed and is in release v0.55.2. Fingers crossed!
